### ISSUE TITLE
Mastodon Landing Page Edit

### ISSUE THREAD

**Issue by alexnya** [mastodon/mastodon#8137]
Hi,

I'm not sure I know where to find what I need to edit on the Mastodon landing page. Here is what I need to edit on the Mastodon landing page:

1) Change Mastodon logo on landing page (on top of login area).
2) Change Mastodon logo on favicon.
3) Change Mastodon logo next to Mastodon text (in center of the page).
4) Replace text below in page (which starts with What is Mastodon?).
5) Remove link to source code in bottom of page.
6) Remove Learn More link.
7) Linking **A look inside** to an intro page.
8) Changing Mastodon elephant image.

Basically I would like to completely customize the Mastodon landing page. I would like detailed instructions on how to complete each of these steps. 

Finally, I have been having an issue where if I put www.mydomainname.com for Mastodon (just as an example), it doesn't redirect me to mydomainname.com. Rather it goes to www.mydomainname.com which is unsecure and I want it to go to mydomainname.com which is secure and has a SSL certificate installed. How do I resolve this?

**Comment by nightpool** [comment]
>Remove link to source code in bottom of page.

This is not possible, as it would be a violation of the AGPL license. You have to provide the source code of your instance to *anyone* who can visit it, via a "prominent" link.

**Comment by alexnya** [comment]
So basically if I modify the source code I have to include a link to the modified source code? Can you explain why that is? I tried looking it up online but I ended up being confused. Also what about the other bits that I need to edit (1-4, 6-8)?

**Comment by trwnh** [comment]
@alexnya The license on the code is the Affero GNU Public License (AGPL), version 3 (v3). Per the terms of the license, source code is made public, distribution of the source code over a network must include a link to this source code, and any modifications and derivatives must also be published under the same AGPL license as well. 
- Wikipedia: https://en.wikipedia.org/wiki/Affero_General_Public_License
- Full license text: https://www.gnu.org/licenses/agpl-3.0.en.html
- Summary / tl;dr: https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)

**Comment by alexnya** [comment]
Which files can I find the parts that I need to edit (1-4, 6-8)?

**Comment by alexnya** [comment]
Also now when I go to my Mastodon URL (idmanninstitute.net) it takes me to an unknown path page (feel free to check out the URL for yourself).

**Comment by trwnh** [comment]
closing this issue as stale, since the "landing page" has been removed. if there are missing customizzations, please file new issues or find existing issues for those
